assets/src/edit-story/components/panels/layerStyle.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Row, Numeric } from '../form';
+import { SimplePanel } from './panel';
+import getCommonValue from './utils/getCommonValue';
+
+const BoxedNumeric = styled(Numeric)`
+  padding: 6px 6px;
+  border-radius: 4px;
+`;
+
+function LayerStylePanel({ selectedElements, onSetProperties }) {
+  const opacity = getCommonValue(selectedElements, 'opacity');
```

Comment:
`opacity` will be `undefined` for existing elements.

This will result in `undefined` being displayed in the input field, and there are prop types warnings about it too

Reply:
I'm wondering if it would make sense to add `default` value param for `getCommonValue`. Thoughts?

Reply:
Maybe, but `opacity` can also be accessed elsewhere, e.g. when doing `opacity / 100`

Reply:
Yep, default value would also need reworking check for multiple. Anyway -- will look into the migration logic.

Reply:
Done in 6a79564, PTAL if that seems the correct way to migrate.

Reply:
Looks good at first glance! 👍 

Can't hurt to add a test for it as well, just like for the other migrations 🙂